packages/core/src/locales/he.ts

```diff
@@ -0,0 +1,125 @@
+import type { $ZodStringFormats } from "../checks.js";
+import type * as errors from "../errors.js";
+import * as util from "../util.js";
+
+const Sizable: Record<string, { unit: string; verb: string }> = {
+  string: { unit: "אותיות", verb: "לכלול" },
+  file: { unit: "בייטים", verb: "לכלול" },
+  array: { unit: "פריטים", verb: "לכלול" },
+  set: { unit: "פריטים", verb: "לכלול" },
+};
+
+function getSizing(origin: string): { unit: string; verb: string } | null {
+  return Sizable[origin] ?? null;
+}
+
+export const parsedType = (data: any): string => {
+  const t = typeof data;
+
+  switch (t) {
+    case "number": {
+      return Number.isNaN(data) ? "NaN" : "number";
+    }
+    case "object": {
+      if (Array.isArray(data)) {
+        return "array";
+      }
+      if (data === null) {
+        return "null";
+      }
+
+      if (Object.getPrototypeOf(data) !== Object.prototype && data.constructor) {
+        return data.constructor.name;
+      }
+    }
+  }
+  return t;
+};
+
+const Nouns: {
+  [k in $ZodStringFormats | (string & {})]?: string;
+} = {
+  regex: "קלט",
+  email: "כתובת אימייל",
+  url: "כתובת רשת",
+  emoji: "אימוג'י",
+  uuid: "UUID",
+  uuidv4: "UUIDv4",
+  uuidv6: "UUIDv6",
+  nanoid: "nanoid",
+  guid: "GUID",
+  cuid: "cuid",
+  cuid2: "cuid2",
+  ulid: "ULID",
+  xid: "XID",
+  ksuid: "KSUID",
+  datetime: "תאריך וזמן ISO",
+  date: "תאריך ISO",
+  time: "זמן ISO",
+  duration: "משך זמן ISO",
+  ipv4: "כתובת IPv4",
+  ipv6: "כתובת IPv6",
+  cidrv4: "טווח IPv4",
+  cidrv6: "טווח IPv6",
+  base64: "מחרוזת בבסיס 64",
+  base64url: "מחרוזת בבסיס 64 לכתובות רשת",
+  json_string: "מחרוזת JSON",
+  e164: "מספר E.164",
+  jwt: "JWT",
+  template_literal: "קלט",
```

Comment:
TBH, not sure how to translate that

EDIT: oh ok, now I see that it's a direct translation of the englsh version